lib/target_mvd_web/controllers/api/v1/target_controller.ex

```diff
@@ -18,14 +19,27 @@ defmodule TargetMvdWeb.API.V1.TargetController do
     current_user_id = PowPlug.current_user(conn).id
     target_params = Map.merge(target_params, %{"user_id" => current_user_id})
 
-    with {:ok, %Target{} = target} <- Targets.create_target(target_params) do
-      conn
-      |> put_status(:created)
-      |> put_resp_header("location", Routes.api_v1_target_path(conn, :show, target))
-      |> render("show.json", target: target)
+    case Targets.create_target(target_params) do
+      {:ok, %Target{} = target} ->
+        conn
+        |> put_status(:created)
+        |> put_resp_header("location", Routes.api_v1_target_path(conn, :show, target))
+        |> render("show.json", target: target)
+
+      {:error, :maximum_reached} ->
+        conn
+        |> put_status(:unprocessable_entity)
+        |> render("target_limit_reached.json")
```

Comment:
Instead of having this really specific view, why don't you add a case to `error.json` that handles error messages not related to changesets? 

something like `render("error.json", %{detail: message})`